packages/content-management/table_list_view_table/src/table_list_view_table.tsx

```diff
@@ -335,9 +343,10 @@ function TableListViewTableComp<T extends UserContentCommonSchema>({
   const initialState = useMemo<State<T>>(
     () => ({
       items: [],
+      hasNoItems: undefined,
       totalItems: 0,
       hasInitialFetchReturned: false,
-      isFetchingItems: false,
+      isFetchingItems: true,
```

Comment:
Why are you defaulting to isFetchingItems is true?

Reply:
That's the state when he table mount: it fetches the items.

We initially had an `useEffect` + `useDebounce`. You correctly removed the effect (https://github.com/elastic/kibana/issues/159507) so we only have one of the 2 but we need to wait `300ms` before having this state set to `true`.